docs/make.jl

```diff
@@ -7,25 +7,35 @@ cp(joinpath(docpath, "Project.toml"), joinpath(assetpath, "Project.toml"), force
 
 include("pages.jl")
 
-mathengine = MathJax3(Dict(:loader => Dict("load" => ["[tex]/require", "[tex]/mathtools"]),
-    :tex => Dict("inlineMath" => [["\$", "\$"], ["\\(", "\\)"]],
-        "packages" => [
-            "base",
-            "ams",
-            "autoload",
-            "mathtools",
-            "require",
-        ])))
+mathengine = MathJax3(
+    Dict(
+        :loader => Dict("load" => ["[tex]/require", "[tex]/mathtools"]),
+        :tex => Dict(
+            "inlineMath" => [["\$", "\$"], ["\\(", "\\)"]],
+            "packages" => [
+                "base",
+                "ams",
+                "autoload",
+                "mathtools",
+                "require",
+            ],
+        ),
+    ))
 
-makedocs(sitename = "JumpProcesses.jl",
+makedocs(
+    sitename = "JumpProcesses.jl",
     authors = "Chris Rackauckas",
     modules = [JumpProcesses],
-    clean = true, doctest = false, linkcheck = true, warnonly = [:missing_docs],
+    clean = true,
+    doctest = false,
+    linkcheck = true,
+    warnonly = [:missing_docs],
```

Comment:
```
makedocs(
    sitename = "JumpProcesses.jl",
    authors = "Chris Rackauckas",
    modules = [JumpProcesses],
    clean = true, doctest = false, linkcheck = true, warnonly = [:missing_docs],
    format = Documenter.HTML(;
        assets = ["assets/favicon.ico"],
        canonical = "https://docs.sciml.ai/JumpProcesses/",
        prettyurls = (get(ENV, "CI", nothing) == "true"),
        mathengine,
    ),
    pages = pages,
)
```

in the most recent commit

Reply:
the unnesting could still be a little better though

Reply:
That is looking better! I think if you can get it to not add the initial and final new lines so both parentheses are kept as originally placed it would be good.

Reply:
```
mathengine = MathJax3(
    Dict(:loader => Dict("load" => ["[tex]/require", "[tex]/mathtools"]),
        :tex => Dict("inlineMath" => [["\$", "\$"], ["\\(", "\\)"]],
            "packages" => [
                "base",
                "ams",
                "autoload",
                "mathtools",
                "require",
            ])))

makedocs(sitename = "JumpProcesses.jl",
    authors = "Chris Rackauckas",
    modules = [JumpProcesses],
    clean = true, doctest = false, linkcheck = true, warnonly = [:missing_docs],
    format = Documenter.HTML(; assets = ["assets/favicon.ico"],
        canonical = "https://docs.sciml.ai/JumpProcesses/",
        prettyurls = (get(ENV, "CI", nothing) == "true"),
        mathengine),
    pages = pages)
```

Reply:
mathengine = MathJax3(Dict(:loader => Dict("load" => ["[tex]/require", "[tex]/mathtools"]), could still be 1 line so might be the calc is slightly off but it's much better now regardless